oxide-openapi-gen-ts/src/client/base.ts

```diff
@@ -21,7 +21,7 @@ export const getSortedSchemas = (spec: OpenAPIV3.Document) => {
     Object.keys(spec.components?.schemas || {}).map((name) => [
       name,
       JSON.stringify(spec.components!.schemas![name])
-        .match(/#\/components\/schemas\/[A-Za-z0-9]+/g)
+        .match(/#\/components\/schemas\/[A-Za-z0-9_]+/g)
```

Comment:
Maybe this could just be like `.+`? or `\w+`. or even `.startsWith('#/components/schemas/')? No need to be this specific to begin with

Reply:
Makes sense to me. I can see if the Open API spec has any guidance on allowed characters.

Reply:
I think this answers it:

> All the fixed fields declared above are objects that MUST use keys that match the regular expression: ^[a-zA-Z0-9\.\-_]+$.